registry/handlers/app.go

```diff
@@ -232,6 +227,83 @@ func (app *App) register(routeName string, dispatch dispatchFunc) {
 	app.router.GetRoute(routeName).Handler(app.dispatcher(dispatch))
 }
 
+func (app *App) configureDriver(configuration *configuration.Configuration) {
+	for {
```

Comment:
Just wondering why we would want to do iterative rewrites. Are there any cases where that's necessary?